### DISCUSSION TITLE
Timers

### DISCUSSION THREAD

**Discussion by alex-savin** [tetele/onju-voice-satellite#76] [General]
Hey Folks,

I see that timers are added to the main branch. Thank you!
May I ask how to use them?

Best,
Alex

**Comment by jhbruhn** [comment]
Tell the Voice Assistant to set a timer for x time-units (seconds, minutes, hours I guess). The exact way depends on the intents language you are using. After the set amount, the Onju Voice will play a chime once.

From my experience, when using LLMs as Assist agent, they will sometimes claim to have set a timer, without actually doing it.
